userproject/Model_Schmuker_2014_classifier_project/README.txt

```diff
@@ -1,66 +1,62 @@
-Author: Alan Diamond, University of Sussex, 2014
+A neuromorphic network for generic multivariate data classification
+===================================================================
 
-This project recreates using GeNN the spiking classifier design used in the paper
+This project recreates the spiking classifier using GeNN as proposed in the paper by 
```

Comment:
Is this description as 'spiking classifier' articulate?

Reply:
I think

> This project recreates the spiking classifier proposed in the paper by Michael Schmuker, Thomas Pfeil and Martin Paul Nawrota using GeNN

would be better

Reply:
Yes, definitely.